packages/cadl-ranch-specs/http/models/inheritance/main.cadl

```diff
@@ -1,60 +1,253 @@
-import "@cadl-lang/rest";
 import "@azure-tools/cadl-azure-core";
 import "@azure-tools/cadl-ranch-expect";
 
 using Cadl.Rest;
 using Cadl.Http;
 using Azure.Core.Foundations;
 
-@doc("Illustrates round-trip polymorphic models.")
-@scenarioService("/polymorphic")
-namespace BasicPolymorphicModels;
+@doc("Illustrates inheritance and polymorphic model.")
+@scenarioService("/models/inheritance")
+namespace Models.Inheritance;
```

Comment:
This is great name! specially as there is also discriminated union in cadl that we'll need to support which will difer from this but is also polymorhism